os/arch/arm/src/amebad/amebad_spi.c

```diff
@@ -675,6 +678,31 @@ int amebad_spi0cmddata(FAR struct spi_dev_s *dev, uint32_t devid, bool cmd)
  ************************************************************************************/
 void amebad_spi1select(FAR struct spi_dev_s *dev, enum spi_dev_e devid, bool selected)
 {
+#ifdef CONFIG_SPI_CS
+	FAR struct amebad_spidev_s *priv = (FAR struct amebad_spidev_s *)dev;
+
+	if (selected == 1){
+		if (devid == 0){
+			//Select cs0, unselect cs1
+			//lldbg("selected cs0\n");
+			gpio_write(&priv->gpio_cs0, 0);
+			gpio_write(&priv->gpio_cs1, 1);
+		}else{
+			//Select cs1, unselect cs0
+			gpio_write(&priv->gpio_cs1, 0);
+			gpio_write(&priv->gpio_cs0, 1);
+		}
+	}else{
+		if (devid == 0){
+			//Unselect cs0
+			//lldbg("unselect cs0\n");
+			gpio_write(&priv->gpio_cs0, 1);
+		}else{
+			//Unselect cs1
+			gpio_write(&priv->gpio_cs1, 1);
+		}
+	}
+#endif
```

Comment:
Could you please add the else case here to support a single slave? We will still need to set the gpio value of a single slave to low or high when selecting or deselecting it. I think that logic is missing here.